packages/jest-cli/src/lib/update_global_config.js

```diff
@@ -60,5 +61,9 @@ export default (globalConfig: GlobalConfig, options: Options): GlobalConfig => {
     newConfig.passWithNoTests = true;
   }
 
+  if ('onlyFailures' in options) {
+    newConfig.onlyFailures = options.onlyFailures || false;
```

Comment:
Do we need to cast onlyFailures to boolean?

Reply:
I did that because of Flow warning.